ctk/common/src/main/kotlin/org/codice/compliance/utils/decorators/IdpPostResponseDecorator.kt

```diff
@@ -69,9 +72,6 @@ class IdpPostResponseDecorator : IdpPostResponse {
 
     val isFormMethodCorrect: Boolean by lazy {
         checkNodeAttribute(responseForm, METHOD, POST)
-                // TODO remove after "DDF form method on Post binding is "post" when it is supposed
```

Comment:
❗️ This change got merged into DDF. Make sure to update DDF in order for tests to pass.